tests/x11/chromium.pm

```diff
@@ -29,12 +29,14 @@ sub run {
     # Additional waiting to prevent unready address bar
     # https://progress.opensuse.org/issues/36304
     wait_still_screen(1);
-    type_string "about:\n";
+    # type_string "about:\n";
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit